mediacapture-streams/MediaStream-GUM.https.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>MediaStream getUserMedia test</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://w3c.github.io/mediacapture-main/#mediastream">
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+
+<p class="instructions">Use a test device with color camera(embedded or external).</p>
+<p class="instructions">When prompted, accept to share your color stream.</p>
+
+<script>
+
+test(function () {
+  assert_true(undefined !== navigator.mediaDevices && undefined !== navigator.mediaDevices.getUserMedia, "navigator.mediaDevices.getUserMedia exists");
+}, "mediaDevices.getUserMedia() is present on navigator");
+
+promise_test(t => {
+  return promise_rejects(t, new TypeError(), navigator.mediaDevices.getUserMedia({}));
+}, "Tests that getUserMedia is rejected with a TypeError when used with an empty options parameter");
+
+promise_test(t => {
+  // Note - integer conversion is weird for +inf and numbers > 2^32, so we
+  // use a number less than 2^32 for testing.
+  return navigator.mediaDevices.getUserMedia({video: {width: {min:100000000}}})
+  .then(stream => {
+    assert_unreached("a Video stream of width 100M cannot be created");
+  })
+  .catch(error => {
+    assert_equals(error.name, "OverconstrainedError", "An impossible constraint triggers a OverconstrainedError");
+    assert_equals(error.constraint, "width", "The name of the not satisfied error is given in error.constraint");
```

Comment:
English nit: "the not satisfied error" => "the unsatisfied constraint"

Reply:
Fixed.